From: Mirsad Todorovac <mtodorovac69@gmail.com>
Subject: mm-vmstat-defer-the-refresh_zone_stat_thresholds-after-all-cpus-bringup-fix.
Date: Wed, 14 Aug 2024 00:03:34 +0200

fix build:

mm/vmstat.c:34:12: error: `vmstat_late_init_done' defined but not used [-Werror=unused-variable]
   34 | static int vmstat_late_init_done;
      |            ^~~~~~~~~~~~~~~~~~~~~

Link: https://lkml.kernel.org/r/b604b19d-c548-4795-adb7-3d0736b8b560@alu.unizg.hr
Fixes: ef534a0d51984 ("mm/vmstat: defer the refresh_zone_stat_thresholds after all CPUs bringup")
Signed-off-by: Mirsad Todorovac <mtodorovac69@gmail.com>
Cc: Saurabh Sengar <ssengar@linux.microsoft.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vmstat.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/mm/vmstat.c~mm-vmstat-defer-the-refresh_zone_stat_thresholds-after-all-cpus-bringup-fix
+++ a/mm/vmstat.c
@@ -31,7 +31,10 @@
 
 #include "internal.h"
 
+#ifdef CONFIG_SMP
 static int vmstat_late_init_done;
+#endif /* CONFIG_SMP */
+
 #ifdef CONFIG_NUMA
 int sysctl_vm_numa_stat = ENABLE_NUMA_STAT;
 
_
